tests/lean/run/discrTreeKey.lean

```diff
@@ -0,0 +1,79 @@
+import Init.Data.Nat.Basic
+import Init.Data.List.Lemmas
+
+/-!
+  This file provides examples of use of the commands #discr_tree_key and #discr_tree_simp_key
+  and guards against any breakage of the commands.
+-/
+
+universe u
+
+open Nat List
+
+/-!
+  We can produce `simp` keys for theorems of the form `=`, `≠`, and `↔` by supplying the name
+  of the declaration.
+-/
+
+#check zero_le
+#discr_tree_simp_key zero_le
```

Comment:
Now that this is moved to `run/`, presumably you intended to put
```
#guard_msgs in
```
before each command whose output you want to test in this file?

Reply:
I'm no longer sure what I'm testing other than it completely breaking. 

I think it's fine if the format of output changes so I don't want to constrain that. 

This seems more like documentation now.

Reply:
I’d include the expected output of each command in this test, using
```
/-- … -/
#guard_msgs in
#discr_tree_simp_key zero_le
```
(I hope you know about the code action that will update the expected output, and its key combination, so that this is not too much work.)

This way, if someone possibly inadvertently changes how the keys are calculated, this test will notify them. And if it is an intentional change, they can easily update it.


But I see your point that then this test isn’t testing your command, it’s now testing the key calculation (with the help of your command). Fair enough, I'll merge this; we can add the test output later if we really want that.